Corona-Warn-App/src/deviceForTesters/AndroidManifest.xml

```diff
@@ -1,21 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<manifest xmlns:tools="http://schemas.android.com/tools"
```

Comment:
This flavored manifest seems to be needless after moving out the provider

Reply:
Good point. Removed :+1: